stable_baselines3/common/base_class.py

```diff
@@ -665,6 +665,9 @@ def load(
         # put other pytorch variables back in place
         if pytorch_variables is not None:
             for name in pytorch_variables:
+                # Skip if PyTorch variable was not defined
```

Comment:
Overall looks good, but can you add bit of clarification here why this might happen and why this is an appropriate fix?
This whole ordeal seems convoluted (variable is expected to be a torch tensor but is not [is a scalar], and somehow it ends up being None?)

Reply:
Good point ;)

Reply:
SAC has an entropy coefficient which can be fixed or optimized.
If it is optimized, we define an additional PyTorch variable `log_ent_coef`, otherwise it is initialized to `None`.

I found a better fix =)

Reply:
nervermind, the earlier fix is needed for backward compatiblity...